docs/client-options.md

```diff
@@ -36,6 +36,9 @@ third party clients.
 ## C
 - [acme-client](https://kristaps.bsd.lv/acme-client/)
 
+## Docker
+- [tls_certificate_generation](https://github.com/leandromoreira/tls_certificate_generation.git)
```

Comment:
This is not the canonical URL, it prompts a redirect to https://github.com/leandromoreira/tls_certificate_generation -- not that it much matters, but there's no need for the `.git` on the end.